_posts/2017-09-08-web.md

```diff
@@ -12,9 +12,84 @@ hierarchy:
 ---
 
 
-## Web apps
+## An Introduction 
 
-TODO
+Welcome to the StartHacking web track! The goal of section is to provide some background information on how the Internet works, as well as provide the conceptual and technical skills needed to provide a successful career in web development.
+
+
+#### How does the Internet work? 
+
+The Internet has become the backbone of our modern-day world, yet few individuals take the time to understand what it is, and how it works. Since it has become such a large part of our lives, a good understanding is needed to become a great Web Developer, and use this tool most effectively.
+
+This section will explain the underlying infrastructure and technologies that make the Internet work; it won't provide a very detailed explanation, and resources will be provided at the end for those who wish to learn more. Any comments, questions, suggestions, etc are welcome and may be directed to the author(s) at [team@hackmit.org](mailto:team@hackmit.org).
+
+Let's take a dive! Let's do a deep dive of exactly what happens, from when you type `https://google.com` in your browser to the moment it shows up on your screen.  
+
+##### Internet Addressing
+
+When the Internet was first founded, it nothing more than a network of computers for academic purposes; there was no way to go to `google.com`, because there was no concept of a named website, or a *domain*. Computers were connected to each other solely on the basis of an address of the form *xxx.xxx.xxx.xxx*, where *x* is a number from *0 - 255*.
```

Comment:
Nit: confused whether each `x` is a number 0-255, or each set of three (`xxx`) is a number 0-255. Same applies below for IPv6

Reply:
also maybe like to [this](https://en.wikipedia.org/wiki/History_of_the_Internet) for further reading